packages/dx-grid-core/src/plugins/grouping-state/helpers.ts

```diff
@@ -1,4 +1,9 @@
-export const adjustSortIndex = (groupingIndex, grouping, sorting) => Math.max(
+import { PureComputed } from '@devexpress/dx-core';
+import { Grouping, Sorting } from '../../types';
+
+export const adjustSortIndex: PureComputed<[number, Grouping[], Sorting[]]> = (
+  groupingIndex, grouping, sorting,
```

Comment:
Can we add a special type for all arguments? That makes intellisense more friendly.

I mean 

```ts
type groupingIndex = number
```